modules/programs/zsh.nix

```diff
@@ -50,6 +50,18 @@ let
         '';
       };
 
+      expireDupsFirst = mkOption {
```

Comment:
Could simply call it `expireDuplicatesFirst`, it's not a huge length difference 🙂